src/component/CreateBook.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import '../CSS/form.css';
+
+const CreateBook = () => {
+  // Define state variables for the book title and author
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+
+  // Handle input changes and update the state variables
+  const handleTitleChange = (event) => {
+    setTitle(event.target.value);
+  };
+
+  const handleAuthorChange = (event) => {
+    setAuthor(event.target.value);
+  };
+
+  // Handle form submission (you can add your logic here)
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    // You can add your code to handle the submission of the book data here
+    // For example, you can send an API request to save the book data
+  };
+
+  return (
+    <div className="form-container">
+      <h2>Add a New Book</h2>
+      <form onSubmit={handleSubmit}>
+        <div>
+          {/* <label htmlFor="title">Title:</label> */}
+          <input
+            type="text"
+            id="title"
+            placeholder="Title"
+            value={title}
+            onChange={handleTitleChange}
+            required
+          />
+        </div>
```

Comment:
Kindly make sure that there is also an input field to add an `author` on the add book form component.

Reply:
- [ ] @Kwamay as previously requested kindly consider adding the input field for the author, It is part of the project requirement, we are supposed to replace the category input with the author input.
![image](https://github.com/Kwamay/React-Books/assets/61408860/0e181b57-9898-451a-b442-4665f153464b)